include/deal.II/meshworker/loop.h

```diff
@@ -247,17 +247,12 @@ namespace MeshWorker
               if (own_cell != own_neighbor && loop_control.faces_to_ghost==LoopControl::never)
                 continue;
 
-              // Deal with
-              // refinement edges
-              // from the refined
-              // side. Assuming
-              // one-irregular
-              // meshes, this
-              // situation should
-              // only occur if
-              // both cells are
-              // active.
-              if (cell->neighbor_is_coarser(face_no))
+              // Deal with refinement edges from the refined side. Assuming one-irregular
+              // meshes, this situation should only occur if both cells are active.
+              const bool periodic_neighbor = cell->has_periodic_neighbor(face_no);
+
+              if ((!periodic_neighbor && cell->neighbor_is_coarser(face_no))
+                  ||(periodic_neighbor && cell->periodic_neighbor_is_coarser(face_no)))
```

Comment:
I stumbled over this for a second -- can you add a space after `||`, or move the second condition onto a line of its own, to make it easier to read? The vertical lines and opening parenthesis just look too similar on my screen :-)